spec/models/tenant_spec.rb

```diff
@@ -416,6 +417,56 @@
       expect { Tenant.switch("typo") { nil } }.to raise_exception(Apartment::TenantNotFound)
       expect { Tenant.switch("notypo") { nil } }.not_to raise_exception
     end
+
+    it "renames the files storage folder when updating the schema" do
+      tenant = create(:tenant)
+
+      expect(tenant).to receive(:rename_storage)
+
+      tenant.update!(schema: "super")
+    end
+  end
```

Comment:
Since tests creating tenants take a few seconds to run, and we're testing the results of this operation in the `#rename_storage` tests, what do you think about removing this test?